### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by jwnpoh**
This PR addresses:

1. typos in the `--revert` help messages, change `it's` to `its`.
2. My earlier PR #147 failed to include the check for `--yes` flag in `deleteProjectCommand`. This PR fixes that.

### CODE REVIEW

File: cli/delete.go [hunk 1]
```diff
@@ -55,9 +55,11 @@ func deleteProjectCommand(t *core.Timetrace) *cobra.Command {
...
 
-			if !askForConfirmation() {
-				out.Info("Record NOT deleted.")
-				return
+			if !confirmed {
+				if !askForConfirmation() {
+					out.Info("Project NOT deleted.")
+					return
+				}
 			}
```
Comment (aligator) [inline cli/delete.go:63]: Wouldn't that be a bit nicer code? Avoid stacked ifs if possible and feasible.
```suggestion
			if !confirmed && !askForConfirmation() {
				out.Info("Project NOT deleted.")
				return
			}
```

**Review by aligator — CHANGES_REQUESTED** [review]
Hi, thanks for the PR. 

I have just a small change request.

File: cli/delete.go (hunk 1 above)
Comment (jwnpoh) [reply cli/delete.go:63]: Yes that makes sense. In fact, I was merely mindlessly copying the if-logic from the `deleteRecordCommand` and did not even think that there was a cleaner way to do it. I will submit another commit with an edit to `deleteRecordCommand` as well.

Comment (jwnpoh) [reply cli/delete.go]: While discussing code style, I also refactored the `if options.Revert` checks to remove the `else` statement for cleaner code, following the advice from [https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88](url).

Please have a look at the new commit and let me know if that's alright!

**Review by dominikbraun — APPROVED** [review]
Nice! The best `else`-blocks are the `else`-blocks that don't exist 😎 

This fix will be included in the next patch release later today.

**Review by aligator — APPROVED** [review]

**Comment by dominikbraun** [comment]
This PR is included in [timetrace v0.11.1](https://github.com/dominikbraun/timetrace/releases/tag/v0.11.1).

**Pull request merged**